packages/grid/x-data-grid/src/components/GridScrollArea.tsx

```diff
@@ -72,6 +73,11 @@ function GridScrollAreaRaw(props: ScrollAreaProps) {
   const ownerState = { ...props, classes: rootProps.classes };
   const classes = useUtilityClasses(ownerState);
 
+  const derivedHeaderHeight = React.useMemo(
+    () => Math.floor(rootProps.headerHeight * densityFactor),
+    [rootProps.headerHeight, densityFactor],
+  );
```

Comment:
Does it worth using a `useMemo`? 
- computation is light (one multiplication) 
- returned value is an integer (so no reference problem)

Reply:
Removed